pkg/util/runtimeconfig/manager_test.go

```diff
@@ -204,8 +209,8 @@ func TestOverridesManager_StopClosesListenerChannels(t *testing.T) {
 			return val, nil
 		},
 	}
-
-	overridesManager, err := NewRuntimeConfigManager(overridesManagerConfig)
+	registry := prometheus.NewRegistry()
```

Comment:
Typically in tests we just pass nil (unless testing metrics), and make sure that New function can accept nil.

Reply:
thanks, done, commit pushed